### DISCUSSION TITLE
Meeting with the Mon/Wed class maintainers today

### DISCUSSION THREAD

**Discussion by jtr13** [jtr13/cc22tt#10] [General]
I'm meeting at 4:10pm today by Zoom with the maintainers from the Mon/Wed section. If you want to say hi, ask any questions, or get a refresher on the PR merging process, you're welcome to join. Please email for the link as I don't want to post it publicly. 
@Jishan13 @SelenaDing0312 @shubhambhalala @naishadhparmar @jl6298 @zcysxy

**Comment by jl6298** [comment]
Dear Mrs Robbin,

Could you send me the zoom link? Thank you.

Best regards,
Jincheng

**Reply by jtr13** [reply]
Hi -- please send me an email (jtr13@columbia.edu) Thanks.

**Comment by Jishan13** [comment]
Professor,
  Can I have the meeting link ?


Thank you.

Yours sincerely,
Jishan Desai

**Reply by jtr13** [reply]
Please send me an email... thx.
